src/rcx/src/gs.cpp

```diff
@@ -64,6 +64,12 @@ gs::~gs()
 void gs::freeMem()
 {
   if (init_ & ALLOCATED) {
+    for (int i = 0; i < pldata_.size(); i++) {
+      pixmap* pm = pldata_[i].plane;
+      if (pm != nullptr) {
+        free(pm);
+      }
```

Comment:
Freeing a nullptr is a no-op so no need for the check.

Reply:
Oh I wasn't sure of that, thank you

Reply:
https://en.cppreference.com/w/c/memory/free  

cppreference.com is helpful for language questions.